horace_core/change_crystal.m

```diff
@@ -1,44 +1,68 @@
 function out = change_crystal(in_data,alignment_info,varargin)
-% Change the crystal lattice and orientation of an sqw object stored in a file
-% or celarray of files
+% Change the crystal lattice and orientation of sqw/dnd object or objects
+% placed in various type of containers.
```

Comment:
```suggestion
% placed in various type of containers. For sqw objects, if -dnd_only is specified, the changes will be made to the dnd
sub-object (.data).
```

Reply:
no, it will fail. Change to `dnd` only part of sqw objects breaks object interity.

Reply:
@abuts I'm looking at lines 82-87 where if it is -dnd_only, if it's an sqw it will change crystal on the .data bit of in_data. I think this is what I described. Happy to accept additional comments so it's clear what is actually going on

Reply:
well -- indeed the cut on sqw object with `-dnd_only` option produces incorrect result which should be fixed.